letsencrypt-auto-source/letsencrypt-auto

```diff
@@ -1656,6 +1656,7 @@ from sys import argv, exit
 from urllib2 import build_opener, HTTPHandler, HTTPSHandler, HTTPError
 
 
+#test
```

Comment:
I'm not in love with leaving the test key in here. Is there a better place we could put it? Maybe git history suffices, or maybe we could at least comment it out so I don't edit this line later and wonder why it doesn't take effect.